code/src/components/Question3.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+const Question3 = ( {onInvestmentTypeChange} ) => {
+ const investmentTypes = [ "Gold", "Bitcoin", "Silver"];
+
+    return (
+        <>       
+        <p>Preferred Investment:</p>
+                 { investmentTypes.map(item => (
+                     <label key={item}>
+                        <input
+                            type="radio"
+                            name="investmentType"
+                            value={item}
+                            onChange={(e) => onInvestmentTypeChange(e)}
+                        />
```

Comment:
You could add the required-attribute here as well. Right now, the user doesn't have to choose a preferred investment.